src/hotspot/cpu/x86/x86_32.ad

```diff
@@ -1813,11 +1813,18 @@ encode %{
       emit_d32_reloc(cbuf, ($meth$$method - (int)(cbuf.insts_end()) - 4),
                      rspec, RELOC_DISP32);
       __ post_call_nop();
-      // Emit stubs for static call.
-      address stub = CompiledStaticCall::emit_to_interp_stub(cbuf);
-      if (stub == NULL) {
-        ciEnv::current()->record_failure("CodeCache is full");
-        return;
+      address mark = cbuf.insts_mark();
```

Comment:
no need to init `address mark` here. 
`emit_to_interp_stub` can do it internally as it worked before

Reply:
It is need for `mark_off`. I'll move it there.

Reply:
Or we can keep it and the code will be the same as in x86_64.ad.
What do you think?